### PR TITLE
Add test for checking issue #420

### LINKED ISSUE #420: cssnano not playing nice with font-display
In my postcss config I've got font-magician configured to add `font-display: optional;`, but there seems to be an issue with cssnano 3.x/4.x;

after getting errors & hoping that `cssnano@next` would clear it up, I've manually added all the plugins & selectively disabled them until no errors occurred:

```js
[
  require('postcss-colormin')(),
  /*
  require('postcss-convert-values')(),
  */
  require('postcss-discard-comments')(),
  require('postcss-discard-duplicates')(),
  require('postcss-discard-empty')(),
  require('postcss-discard-overridden')(),
  /*
  require('postcss-discard-unused')(),
  */
  require('postcss-merge-idents')(),
  require('postcss-merge-longhand')(),
  require('postcss-merge-rules')(),
  require('postcss-minify-font-values')(),
  require('postcss-minify-gradients')(),
  require('postcss-minify-params')(),
  require('postcss-minify-selectors')(),
  require('postcss-normalize-charset')(),
  require('postcss-normalize-display-values')(),
  require('postcss-normalize-positions')(),
  require('postcss-normalize-repeat-style')(),
  /*
  require('postcss-normalize-string')(),
  */
  require('postcss-normalize-timing-functions')(),
  require('postcss-normalize-unicode')(),
  /*
  require('postcss-normalize-url')(),
  require('postcss-normalize-whitespace')(),
  */
  require('postcss-ordered-values')(),
  require('postcss-reduce-idents')(),
  require('postcss-reduce-initial')(),
  require('postcss-reduce-transforms')(),
  require('postcss-svgo')(),
  require('postcss-unique-selectors')(),
  require('postcss-zindex')(),
]
```

the errors that occur are either `TypeError: value.charCodeAt is not a function` or `TypeError: node.value.replace is not a function` and only occur if the `@font-family` blocks have a `font-display` declaration present, i.e. `font-display: optional ;`

### PR DESCRIPTION
**Pull request by kuflash**
Test for checking integration with `postcss-font-magician` with `display` parameter.

fixes #420

### CODE REVIEW

**Comment by kuflash** [comment]
Now it's failed, but with new release `postcss-font-magician` should pass successfully

**Comment by evilebottnawi** [comment]
@kuflash thanks!

File: packages/cssnano/package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "cross-env": "^5.1.6",
     "cssnano-preset-advanced": "^4.0.0-rc.2",
     "ncp": "^2.0.0",
-    "postcss-font-magician": "^2.0.0",
+    "postcss-font-magician": "2.2.0",
```
Comment (evilebottnawi) [inline packages/cssnano/package.json]: please use `^`

**Comment by evilebottnawi** [comment]
@kuflash looks yarn broken :confused: let't try restart

**Comment by evilebottnawi** [comment]
@kuflash oh, `postcss-font-magician` require `nodejs@6`, thanks for PR. I will drop `nodejs@4` tomorrow and merge you PR, on other version all works fine :+1:

**Comment by kuflash** [comment]
@evilebottnawi thank you!

**Pull request merged**